sdk/cargo-test-bpf/src/main.rs

```diff
@@ -43,13 +47,13 @@ impl Default for Config {
     }
 }
 
-fn spawn<I, S>(program: &Path, args: I)
+fn spawn<I, S>(program: &Path, args: I, generate_rerun: bool)
 where
     I: IntoIterator<Item = S>,
     S: AsRef<OsStr>,
 {
     let args = args.into_iter().collect::<Vec<_>>();
-    print!("Running: {}", program.display());
+    print!("CARGO-TEST-BPF Running: {}", program.display());
```

Comment:
Can this hunk be reverted?  It seems a little yelly

Reply:
I added this because cargo  and cargo-build-bpf output their own "Running: " message, and it's annoying to figure out which of the several child processes in a complicated build system is running the printed command, but ok, I'll revert it.

Reply:
I see. I think the yelling put me off. Childhood flashbacks perhaps, how about this instead:
```rust
print!("cargo-test-bpf child: {}", program.display());
```

Reply:
caps help me to quickly see in a stream of text on terminal the message i'm looking for, but I understand the sentiment :-) i'll lower the case

Reply:
Welp that an arms race and everybody ends up yelling in the end :)